config/vufind/contentsecuritypolicy.ini

```diff
@@ -0,0 +1,32 @@
+; Settings for Content Security Policy header; you can learn more here:
+; https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+[CSP]
+; It is possible to use CSP in report-only mode. See report_uri setting below.
+; This setting could be set differently for each APPLICATION_ENV value (written as
+; array key below). Please note, that Whoops error handler (enabled in development
+; mode) does not show correctly when report_only is false.
+report_only[production] = false
+report_only[development] = true
+
+; The nonce (number used once) - unique number for each request, it is strongly
+; recommend to keep this setting on
+use_nonce = true
+
+; Directives, you can find available directives, on this page:
+; https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+; For evaluation of CSP you can use this tool provided by Google:
+; https://csp-evaluator.withgoogle.com/
+[Directives]
+default-src[] = "'none'"
+script-src[] = "'strict-dynamic'"
+script-src[] = "'unsafe-inline'"
+script-src[] = "http:"
+script-src[] = "https:"
+connect-src[] = "'self'"
+style-src[] = "'self'"
+img-src[] = "'self'"
```

Comment:
Is this very string img-src setting going to block direct cover URLs as being developed in #1566? I'm wondering if we need to loosen this up by default, and then recommend that people make it more strict if necessary.

Reply:
It i going to block direct cover url.... I do think we could add an example (as I did for MapSelection now) and a note to ObakyKnih setting in ini files... I'll to that on #1566, when this is merged. Is that OK from your point of view?

Reply:
Yes, that makes sense to me!

Reply:
I've added a TODO checkbox on #1566 so we don't forget to do this.